script-library/git-from-src-debian.sh

```diff
@@ -18,13 +19,30 @@ if [ "$(id -u)" -ne 0 ]; then
     exit 1
 fi
 
-if [ "${GIT_VERSION}" = "latest" ] || [ "${GIT_VERSION}" = "lts" ] || [ "${GIT_VERSION}" = "current" ]; then
-    RECENT_TAGS=$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/git/git/tags")
-    GIT_VERSION=$(echo ${RECENT_TAGS} | grep -oE 'name":\s*"v[0-9]+\.[0-9]+\.[0-9]+"' | head -n 1 | sed 's/^name":\s*"v\(.*\)"$/\1/')
-fi
-
 export DEBIAN_FRONTEND=noninteractive
 
+# Source /etc/os-release to get OS info
```

Comment:
Why allow an option to pull from a `ppa` if this script is called `git-from-src-debian.sh`?

Reply:
See #511. There is a very specific circumstance where this works - ubuntu and you choose latest.  Everything else you need to build.  We could rename the script, but that breaks links.  I suppose we could default this to not use the PPA, however.

Reply:
Tweaked default.